sdks/python/apache_beam/utils/annotations.py

```diff
@@ -90,6 +90,8 @@ def inner(*args, **kwargs):
       message = '%s is %s' % (fnc.__name__, label)
```

Comment:
Should `extra_message` be actually just the `message` that can fully customize the message?